README.md

```diff
@@ -116,107 +160,51 @@ You can also download raw OpenAPI [definitions](https://github.com/qdrant/qdrant
 
 For faster production-tier searches, Qdrant also provides a gRPC interface. You can find gRPC documentation [here](https://qdrant.tech/documentation/quick_start/#grpc).
 
-### Clients
-
-Qdrant offers the following client libraries to help you integrate it into your application stack with ease:
-
-- [Python client](https://github.com/qdrant/qdrant-client)
-- [Go client](https://github.com/qdrant/go-client)
-- [Rust client](https://github.com/qdrant/rust-client)
-- [JavaScript/TypeScript client](https://github.com/qdrant/qdrant-js)
-
 ## Features
 
 ### Filtering and Payload
 
-Qdrant supports any JSON payload associated with vectors. It does not only store payload but also allows filter results based on payload values.
-It allows any combinations of `should`, `must`, and `must_not` conditions, but unlike ElasticSearch post-filtering, Qdrant guarantees all relevant vectors are retrieved.
+Qdrant enables JSON payloads to be associated with vectors, providing both storage and filtering based on payload values. It supports various combinations of `should`, `must`, and `must_not` conditions, ensuring retrieval of all relevant vectors unlike `ElasticSearch` post-filtering.
```

Comment:
Getting all relevant points relates to qdrant's hybrid search, rather than to arranging conditions on the filter

Reply:
Good point! Will address in the following PR!